src/components/WithSidebar/index.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
```

Comment:
Esp. since this is a new component, docstrings explaining what these are and how to instantiate. Please and thanks.

Reply:
fair fair.  I am mostly planning a big wave of docstring adds as part of the next PR, but can add for this as it got re-worked.